### DISCUSSION TITLE
dladdr1 not found

### DISCUSSION THREAD

**Discussion by jusn6935** [ptitSeb/box86#538] [Q&A]
I am getting a PltResolver Error. I wrote a small program in LiveCode and it works great on Ubuntu 32bit (ver16.04lts) but I get an error when I try to run in Raspbian (Raspbian GNU/Linux 10 (buster)) 32bit (RPi 3B+). I have install the Box86 software several times, without success. I have tried to research the issue (https://manned.org/man/debian-buster/dladdr1) but this is beyond my comprehension.
The error is as follows:
Error: PltResolver: Symbol dladdr1(ver 14: dladdr1@GLIBC_2.3.3) not found, cannot apply R_386_JMP_SLOT 0x8dcd150 (0x8094466) in /home/pi/Desktop/32/Maw_Interface.

Thx a Bunch

**Comment by ptitSeb** [comment]
This function "dladdr1" will be tricky to fully support. I can add a version that ignore the extra parameter (so acting as "dladdr"), maybe it's enough?

**Reply by jusn6935** [reply]
The program is nothing elaborate just was trying to better understand LiveCode and see if I wanted to continue learning it. If the solution is to time intensive I could look into doing it in python or some other language. Don't spend a lot of time on it, not that impotant.  Issue file - https://github.com/jusn6935/Maw_Interface
 Thx a bunch (HNY)
